src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/config/ResourceServerConfiguration.kt

```diff
@@ -15,6 +15,7 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
     http
       .sessionManagement()
       .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+      .and().headers().frameOptions().sameOrigin()
```

Comment:
Had to include this because the H2 console uses iframes to display each of the panes